docs/cluster-management/security.md

```diff
@@ -68,4 +72,4 @@ In order to connect to the operator via the CLI, you must provide valid AWS cred
 
 All APIs are accessible via HTTPS (in addition to HTTP). The SSL certificate is autogenerated during installation using `localhost` as the Common Name (CN). Therefore, clients will need to skip certificate verification (e.g. `curl -k`) when using HTTPS.
 
-To use AWS's default (trusted) certificate or your own certificate, you can [configure API Gateway](../guides/api-gateway.md) as a proxy to your Cortex cluster. If you do this, you'll also likely want to set Cortex's API load balancer to be internal (versus internet-facing, which is the default); do this by setting `api_load_balancer_scheme: internal` in your [cluster configuration](config.md) file before creating your cluster (e.g. via `cortex cluster up -c cluster.yaml`).
+To use AWS's default (trusted) certificate or your own certificate, you can [set up API Gateway](../guides/api-gateway.md) to be a proxy to your Cortex cluster. Since the API load balancer that Cortex creates is internet-facing by default, you will also need to set `api_load_balancer_scheme: internal` in your [cluster configuration](config.md) file (before creating your cluster) in order to force traffic to go through your API Gateway endpoint.
```

Comment:
`Since the API load balancer that Cortex creates is` -> `Since the API load balancer created by Cortex is`